google/api_core/future/base.py

```diff
@@ -16,11 +16,8 @@
 
 import abc
 
-import six
```

Comment:
I noticed that we still have a `six` pin in `testing/constraints-3.6.txt`, let's remove that, too.

Reply:
6e21b01